tdrs-frontend/src/actions/reports.js

```diff
@@ -16,112 +16,118 @@ export const SET_FILE_LIST = 'SET_FILE_LIST'
 export const FETCH_FILE_LIST_ERROR = 'FETCH_FILE_LIST_ERROR'
 export const DOWNLOAD_DIALOG_OPEN = 'DOWNLOAD_DIALOG_OPEN'
 
-export const clearFile = ({ section }) => (dispatch) => {
-  dispatch({ type: CLEAR_FILE, payload: { section } })
-}
+export const clearFile =
```

Comment:
The prettier update caused some linter errors on this file so I used `eslint src/ --fix` to get this file to pass the linter.

Reply:
Maybe it's just how github is rendering it or my unfamiliarity with JS but does the indentation seem off?

Reply:
I think its github, it looks right in my IDE and the formatting matches what https://prettier.io/playground gives as well.